Don't reset the auth token when sStartAutomatically flag is set (#668)
microsoft/appcenter-sdk-react-native: CHANGELOG.md

* **[Feature]** Catch low memory warning and provide the API to check if it has happened in last session:  `Crashes.hasReceivedMemoryWarningInLastSession()`.

### App Center Auth

> * **[Fix]** Fix silently signed out on application restart if auth module isn't initialized before other modules.

Reviewer: I believe the problem only happens when auth module is initialized **later** than other modules.
```suggestion
* **[Fix]** Fix silently signed out on application restart if appcenter-auth module is initialized after other appcenter modules.
```
